reference/configuration/kernel.rst

```diff
@@ -308,6 +308,56 @@ the configuration options used to run the application. This allows for example
 to run an application with the ``prod`` config (``kernel.environment``) in different
 scenarios like ``staging`` or ``production`` (``kernel.runtime_environment``).
 
+``kernel.runtime_mode``
+-----------------------
+
+**type**: ``string`` **default**: ``%env(query_string:default:container.runtime_mode:APP_RUNTIME_MODE)%``
+
+This parameter stores a query string of the current runtime mode used by the
+application. For example, the query string looks like ``web=1&worker=0`` when
+the application is running in web mode and ``web=1&worker=1`` when running in
+a long-running web server. This parameter can be set by using the
+``APP_RUNTIME_MODE`` env var.
+
+.. versionadded:: 6.4
+
+    The ``kernel.runtime_mode`` parameter was introduced in Symfony 6.4.
+
+``kernel.runtime_mode.web``
+---------------------------
+
+**type**: ``boolean`` **default**: ``%env(bool:default::key:web:default:kernel.runtime_mode:)%``
```

Comment:
```suggestion
**type**: ``boolean`` **default**: ``%env(bool:default::key:web:default:kernel.runtime_mode??)%``
```

Reply:
Is this correct?

Reply:
I took this from https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/HttpKernel/Kernel.php#L574

This new syntax was introduced here if I am right: https://github.com/symfony/symfony/pull/52079/files#r1363574735